File: src/core/nativos/matematicas.js
export function sumar(a, b) {
    return a + b;
}

export function promedio(lista) {
    if (lista.length === 0) return 0;
    const total = lista.reduce((a, b) => a + b, 0);
    return total / lista.length;
}

export function potencia(base, exponente) {
    return Math.pow(base, exponente);
}

File: src/core/nativos/matematicas.py
import math


def sumar(a, b):
    """Suma dos valores."""
    return a + b


def promedio(valores):
    """Calcula el promedio de una lista de valores."""
    return sum(valores) / len(valores) if valores else 0


def potencia(base, exponente):
    """Eleva una base a un exponente."""
    return math.pow(base, exponente)
